src/config/kubefile.rs

```diff
@@ -104,6 +107,91 @@ pub struct ContextConf {
     pub user: String,
 }
 
+#[derive(Debug, Deserialize, Clone)]
+#[allow(non_snake_case)]
```

Comment:
are you using this so that the deserialize finds the fields? if so, rather do something like this:
```rust
#[derive(Debug, Deserialize, Clone)]
pub struct Exec {
    #[serde(rename = "apiVersion")]
    api_version: String,
    pub args: Option<Vec<String>>,
    pub command: Option<String>,
    pub env: Option<Vec<Env>>,
    pub token: RefCell<Option<String>>,
    pub expiry: RefCell<Option<DateTime<Utc>>>,
}
```